tests/src/ExistingSite/ShowcaseExistingSiteCreatePersonTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_showcase\ExistingSite;
+
+use Drupal\file\Entity\File;
+use Drupal\media\Entity\Media;
+use Drupal\Tests\media\Traits\MediaTypeCreationTrait;
+use Drupal\Tests\TestFileCreationTrait;
+
+/**
+ * Class to test Person content type on existing site tests.
```

Comment:
We need to improve when we can on existing code and comments. There's no value in most of the text of this comment.
```suggestion
 * Tests the person content type.
```